### PR TITLE
Clarify, fix typos

### PR DESCRIPTION
**Pull request by fredster33**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -60,7 +60,7 @@ All this mod does is parallelise each of these loops these loops. The vast major
 
 Each one of the above loops is paralelised, and each one is toggleable.
 
-To make this parallelisation work, there are a lot of patches to the minecraft core code to avoid concurrent access to non-concurrent objects (curse ye fastutil) or to replace them with a working concurrent alternative.
+To make this parallelisation work, there are a lot of patches to the Mnecraft core code to avoid concurrent access to non-concurrent objects (curse ye fastutil) or to replace them with a working concurrent alternative.
```
Comment (Naheulf) [inline README.MD:63]: Mnecraft ?

**Comment by Naheulf** [comment]
If you fix typos, you forget two at line 59:
> All this mod does is parallelise each of these loops ***these loops***. The vast majority of the performance (at least in my testing) was gained from parallelising entit***i***es.

File: README.MD (hunk 1 above)
Comment (fredster33) [reply README.MD]: Fixed, thanks!

**Comment by fredster33** [comment]
> If you fix typos, you forget two at line 59:
> 
> > All this mod does is parallelise each of these loops _**these loops**_. The vast majority of the performance (at least in my testing) was gained from parallelising entit_**i**_es.

@Naheulf Fixed!

**Pull request closed without merging**